src/index.html

```diff
@@ -11,8 +11,57 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product"
+      data-qa="card"
+    >
+      <div class="product-card">
+        <h1 class="product-card__content">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="product-card__content_code">Product code: 195434</p>
+
+        <div class="product-card__reviews">
+          <div class="stars">
+            <ul class="stars__product-rating">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
```

Comment:
Add the modifier class
```suggestion
            <ul class="stars stars--4">
              <li class="stars__star"></li>
              <li class="stars__star"></li>
              <li class="stars__star"></li>
              <li class="stars__star"></li>
              <li class="stars__star"></li>
```